backend/src/apiserver/client_manager.go

```diff
@@ -37,6 +37,10 @@ import (
 const (
 	minioServiceHost       = "MINIO_SERVICE_SERVICE_HOST"
 	minioServicePort       = "MINIO_SERVICE_SERVICE_PORT"
+	minioServiceRegion     = "MINIO_SERVICE_REGION"
+	minioServiceSecure     = "MINIO_SERVICE_SECURE"
+	pipelineBucketName     = "MINIO_PIPELINE_BUCKET_NAME"
```

Comment:
Any reason you add this environment?  Does config override work for you?  If so, it's unnecessary to add a new env for same field

Reply:
true. this is more for consistency.  Personally, I feel we shld remove all the `MINIO_*` and stick with what `viper` provides us.

Reply:
The only reason i prefer `env` over `config` is just it takes less effort than attaching a configmap.